protocol/xmrmaker/checks.go

```diff
@@ -171,3 +172,29 @@ func (s *swapState) checkContract(txHash ethcommon.Hash) error {
 
 	return nil
 }
+
+func (s *swapState) checkAndSetTimeouts(t0, t1 *big.Int) error {
```

Comment:
Can we add a short description of what we are checking and why?  I'm not sure it will be immediately obvious to code auditors.

Reply:
yes, responded and added a comment here

Reply:
Here's another possible description.  Pick and chose what you like out of it:
```
// checkAndSetTimeouts validates and sets our local timeout values. The XMR
// taker, when initiated the swap, must set the timeout to an exact value that
// depends on the network and environment. We also verify that the XMR taker
// didn't wait an excessive amount of time after locking the ETH asset, but
// before telling us.

```